
Just some missing pivot_root.S inclusions ...

  - Rene Rebe <rene@exactcode.de>

--- dietlibc-0.28/arm/dyn_syscalls.S	2004-12-13 11:21:31.000000000 +0100
+++ dietlibc-0.28-fixed/arm/dyn_syscalls.S	2005-02-03 21:26:56.431882216 +0100
@@ -107,6 +107,7 @@
 #include "../syscalls.s/pause.S"
 #include "../syscalls.s/personality.S"
 #include "../syscalls.s/pipe.S"
+#include "../syscalls.s/pivot_root.S"
 #include "../syscalls.s/poll.S"
 #include "../syscalls.s/query_module.S"
 #include "../syscalls.s/init_module.S"
--- dietlibc-0.28/i386/dyn_syscalls.S	2005-01-28 17:04:54.000000000 +0100
+++ dietlibc-0.28-fixed/i386/dyn_syscalls.S	2005-02-03 21:25:58.067754912 +0100
@@ -60,6 +60,7 @@
 #include "../syscalls.s/mprotect.S"
 #include "../syscalls.s/nanosleep.S"
 #include "../syscalls.s/pipe.S"
+#include "../syscalls.s/pivot_root.S"
 #include "../syscalls.s/poll.S"
 #include "../syscalls.s/readv.S"
 #include "../syscalls.s/stat.S"
